content/collections.md

```diff
@@ -0,0 +1,287 @@
+
+To write data back to the server, you use a *method*, the subject of the "methods and forms" article.
+
+### Local Collections
+
+There is a third way to use a collection in Meteor. On the client or server, if you create a collection but do *not* give it a name:
+
+```js
+Scratch = new Mongo.Collection(null);
+```
+You create what's known as a *local collection*. This is a Minimongo collection that has no database connection (ordinarly a named collection would either be directly connected to the database on the server, or via a publication on the client). 
+
+A local collection is simply a convienent way to use the full power of the Minimongo library for in-memory storage. For instance, you might use it instead of a simple array if you'll need to sophisticated queries over your set of data. Or you may want to take advatange of it's *reactivity* on the client to drive some UI in a way that feels natural in Meteor.
+
+## Defining Collections with a Schema
+
+Although Mongo is a schema-less database, which allows maximum flexibility in data structuring, it is generally good practice to use a schema to constraint the contents of your collection to conform to a known format. If you don't, then you tend to end up needing to write defensive code to check and confirm the structure of your data as it *comes out* of the database, instead of when it *goes into* the database. As in most things, you tend to *read things more often than you write them*, and so it's usually easier, and less buggy to use a schema.
+
+In Meteor, the pre-eminent schema package is [aldeed:simple-schema](http://atmospherejs.com/aldeed/simple-schema). It's an expressive, Mongo based schema that's used to insert and update documents.
+
+To write a schema using `simple-schema`, you can simply create a new instance of the `SimpleSchema` class:
+
+```js
+Lists.schema = new SimpleSchema({
+  name: {type: String},
+  incompleteCount: {type: Number, defaultValue: 0},
+  userId: {type: String, regEx: SimpleSchema.RegEx.Id, optional: true}
+});
+```
+
+In this example, from the Todos app, we are doing a few things that are interesting:
+
+1. We attach the schema to the namespace of `Lists` directly. This allows us to check things against the schema of a list directly (rather than via inserting into the DB, see below), such as in a form (see the forms article).
+2. We specify that the `name` field of a list must be a string, and must exist.
+3. We specify the `incompleteCount` is a number, which on insertion is set to `0` if not otherwise specified.
+4. We specify that the `userId`, which is optional, must be a string matching an id regular expression.
+
+You can see from this example, that with relatively little code we've managed to restrict the format of a list significantly.
```

Comment:
This seems like a natural place to link to the docs for simple-schema to show what other restrictions are possible.